Changes

```diff
@@ -99,6 +99,13 @@ Working version
 * #10775: Half-precision floating-point elements in Bigarray.
   (Anton Yabchinskiy, review by Xavier Leroy and Nicolás Ojeda Bär)
 
+- #11563: Add the Dynarray module to the stdlib. Dynamic arrays are
+  arrays whose length can be changed by adding or removing elements at
+  the end, similar to 'vectors' in C++ or Rust.
```

Comment:
Is it enacted that `Dynarrays` will be unboxed soon like C++/Rust vectors? Otherwise it is misleading to mention vectors.

Reply:
No. But there is already a caveat about this in the documentation, I think that leaving the changes entry as-is is fine.